packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorAccountTabs.jsx

```diff
@@ -95,14 +96,16 @@ const DumbKonnectorAccountTabs = props => {
           <DataTab konnector={konnector} trigger={initialTrigger} flow={flow} />
         )}
         {tab === 1 && (
-          <ConfigurationTab
-            konnector={konnector}
-            account={account}
-            flow={flow}
-            addAccount={addAccount}
-            onAccountDeleted={onAccountDeleted}
-            showNewAccountButton={showNewAccountButton}
-          />
+          <HarvestVaultProvider>
+            <ConfigurationTab
+              konnector={konnector}
+              account={account}
+              flow={flow}
+              addAccount={addAccount}
+              onAccountDeleted={onAccountDeleted}
+              showNewAccountButton={showNewAccountButton}
+            />
+          </HarvestVaultProvider>
```

Comment:
Could HarvestVaultProvider be at the route level ? I don't see why it should only wrap the ConfigurationTab ?

Reply:
I guess it makes sense. So you mean wrapping here typically? https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/components/Routes.jsx#L22

Reply:
Yes.

Reply:
Done in c753a08